src/tests/unit/platform/test_alertsv7_api.py

```diff
@@ -1225,6 +1229,23 @@ def test_alert_subtype_watchlistalert_string_class(cbcsdk_mock):
     assert isinstance(alert, WatchlistAlert)
 
 
+def test_watchlistalert_getwatchlistobjects(cbcsdk_mock):
+    """Test WatchlistAlert get_watchlist_objects()."""
+    cbcsdk_mock.mock_request("GET",
+                             "/api/alerts/v7/orgs/test/alerts/f6af290d-6a7f-461c-a8af-cf0d24311105",
+                             GET_ALERT_v7_WATCHLIST_RESPONSE)
+    cbcsdk_mock.mock_request("GET",
+                             "/threathunter/watchlistmgr/v3/orgs/test/watchlists/mnbvc098766HN60hatQMQ",
+                             GET_WATCHLIST_OBJECT_RESP)
+
+    api = cbcsdk_mock.api
+    watchlist_alert = api.select("WatchlistAlert", "f6af290d-6a7f-461c-a8af-cf0d24311105")
+    watchlist_objects = watchlist_alert.get_watchlist_objects()
+    assert isinstance(watchlist_objects, list)
+    assert watchlist_objects[0].__module__ == "cbc_sdk.enterprise_edr.threat_intelligence" and \
```

Comment:
This works but you also could import the Watchlist object then do an `isinstance(watchlist_objects[0], Watchlist)` otherwise if we ever adjusted a package name or something in the future it breaks this test